app/components/post-search-results.js

```diff
@@ -0,0 +1,136 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { getQueryString } from '@wikia/ember-fandom/utils/url';
+
+import config from '../config/environment';
+
+// TODO: Remove this when all discussions' posts are in the index
+const QUIZZES_WHITELIST = [
```

Comment:
I don't have enough context about why this array is needed, but it seems to me like good candidate to replace with some wg variable or if you think that this is overkill then at least to move it to separate file/module together with some function that checks if the request is in context of one of these wikis

Reply:
This is very temporary and will be removed in ~2weeks or so.

Reply:
I'd say that even moving to separate file is an overkill. We intend to enable the new search feature only on quizzes communities at first (because we only index quizzes first) and then after we finish adding all the discussions' posts, we're going to remove this variable completely.